cypress/integration/AnnualPlan/ContentBlockAPButton.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Tests for AP button on a content block
+
+  Scenario Outline: Web site visitor views a page with a button on desktop, tablet or mobile
+    Given user is navigating to "<url>"
+    And screen breakpoint is set to "<screenBreakpoint>"
+    Then the button to download At a Glance should appear at the bottom of the page
+    And a PDF icon located at "profiles/custom/cgov_site/themes/custom/cgov/cgov_common/dist/images/sprites/svg-sprite.svg" should be displayed with a text "Annual Plan & Budget Proposal At A Glance"
```

Comment:
please add a " /" before the URl - the baseUrl value should be without trailing / in the end, and that will break the test - since the correct imageUrl pulled as "/profiles/custom..."